ui/cypress/e2e/explorer.test.ts

```diff
@@ -99,7 +99,7 @@ describe('DataExplorer', () => {
           })
       })
     })
-    it('should put input field in error status when "10" gets one character deleted and becomes "1"', () => {
+    it('should put input field in error status and stay in error status when input is invalid or empty', () => {
```

Comment:
is it possible to combine these two tests, or make the descriptions of them unique?

at a glance, since they all have the same description, it kind of seems like they're duplicates. a little bit of specificity would be helpful here

Reply:
One test is for Histogram bin sizes, which was compacted as part of https://github.com/influxdata/influxdb/pull/15855 - just that the test description was not updated until now.

The other test is for Heat Map bin sizes. The input there has slightly different behavior but we want to test for the same thing essentially, which is that empty and invalid input will put the component into error status. Here, both the test description and the tests were changed.

Reply:
So I believe the test descriptions are fine. And per watts, the nested ``then``s are unnecessary. I will be removing them.